src/order/hom/lattice.lean

```diff
@@ -234,12 +234,16 @@ section boolean_algebra
 variables [boolean_algebra α] [boolean_algebra β] [bounded_lattice_hom_class F α β] (f : F)
 include β
 
-lemma map_compl (a : α) : f aᶜ = (f a)ᶜ := (is_compl_compl.map _).compl_eq.symm
+/-- Special case of `map_compl` for boolean algebras. -/
+lemma map_compl' (a : α) : f aᶜ = (f a)ᶜ := (is_compl_compl.map _).compl_eq.symm
```

Comment:
I don't see any downstream changes to use the primed lemmas, should we still keep them?

Reply:
There is one in `algebra.ring.boolean_ring`. The problem is that [`bounded_lattice_hom_class.to_biheyting_hom_class`](https://github.com/leanprover-community/mathlib/pull/15308/files#diff-b6b7114374d685445215c36d39277ce4cee33da097965b08e6d791d0fa75483fR141-R143) can't be an instance in Lean 3 because it would loop.